log10/feedback/feedback.py

```diff
@@ -70,6 +73,52 @@ def list(self, offset: int = 0, limit: int = 50, task_id: str = "") -> httpx.Res
                 logger.error(e.response.json()["error"])
             raise
 
+    def list_v2(self, page: int = 1, limit: int = 50, task_id: str = "", filter: str = "") -> httpx.Response:
+        query = """
+        query OrganizationFeedback($id: String!, $filter: String, $taskId: String, $page: Int, $limit: Int) {
+            organization(id: $id) {
+                id
+                feedbackV2(filter: $filter, taskId: $taskId, page: $page, limit: $limit) {
+                pageInfo{
+                    totalCount
+                    totalCount
+                    currentPage
+                }
+                nodes {
+                    id
+                    jsonValues
+                    task {
+                    id
+                    name
+                    }
+                    completions {
+                    id
+                    }
+                }
+                }
+            }
+        }
+        """
+
+        variables = {
+            "id": self._log10_config.org_id,
+            "taskId": task_id or None,
```

Comment:
The default value for `task_id` is empty string, is there significance to `None` or does it just need to be falsey? Another approach is to default it to `None`. (Just something that jumped out from a readability standpoint)